inst/uvhydrograph/uvhydrograph.Rmd

```diff
@@ -28,6 +28,7 @@ output:
 ***
 
 ```{r echo=FALSE, warning=FALSE, message=FALSE, results='asis'}
+library(lubridate)
```

Comment:
We should probably investigate more to see why this is needed. It shouldn't have to be explicitly stated here